src/Runner.jl

```diff
@@ -109,6 +112,16 @@ function generate_compiler_wrappers!(platform::Platform; bin_path::AbstractStrin
             write(io, "export $(name)=\"$(val)\"\n")
         end
 
+        if length(unsafe_flags) >= 1
+            write(io, """
+            if [[ \"\$@\" =~ \"$(join(unsafe_flags, '|'))\" ]]; then
+                echo \"You used one or more of the unsafe flags: $(join(unsafe_flags, ", "))\"
```

Comment:
```suggestion
                echo \"You used one or more of the unsafe flags: $(join(unsafe_flags, ", "))\\nRepent.\"
```

Reply:
Also, can you `>&2` it so that it's red?